<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QPushButton" name="RedButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Red Button</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="BlueButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Blue Button</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StartGameButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Game</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="RoundProgressBar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Simon Says Game!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>-500</x>
       <y>-90</y>
       <width>1611</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/pics/RedBlueSimonBackground.png);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="loserText">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>LOSER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>imageLabel</zorder>
    <zorder>RedButton</zorder>
    <zorder>BlueButton</zorder>
    <zorder>StartGameButton</zorder>
    <zorder>RoundProgressBar</zorder>
    <zorder>Title</zorder>
    <zorder>loserText</zorder>
   </widget>
   <widget class="QMenuBar" name="menubar_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>800</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>3</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
